__tests__/app.test.js

```diff
@@ -407,4 +407,90 @@ describe("7.getUsers", () => {
     })
 })
 
+// refer back to line 60 for getting reviews without queries.
+describe("8.getReviews using queries", () => {
+    describe("GET /api/reviews queries", () => {
+        test("returns reviews queried with only a sortOn parameter in default DESC order", () => {
+            return request(app)
+            .get("/api/reviews?sortOn=votes")
+            .expect(200)
+            .then(res => {
+                let body = res.body.reviews;
+                expect(body).toBeSortedBy('votes', {descending: true});
+                expect(Array.isArray(body)).toBe(true);
+                expect(body.length).toBe(13);
+            })
+        })
+        test("returns reviews queried with sortOn and order parameters", () => {
+            return request(app)
+            .get("/api/reviews?sortOn=votes&order=ASC")
+            .expect(200)
+            .then(res => {
+                let body = res.body.reviews;
+                expect(body).toBeSortedBy('votes');
+                expect(Array.isArray(body)).toBe(true);
+                expect(body.length).toBe(13);
+            })
+        })
+        test("returns relevant reviews queried with only a category parameter (default created_at DESC order)", () => {
+            return request(app)
+            .get("/api/reviews?category=social+deduction")
+            .expect(200)
+            .then(res => {
+                let body = res.body.reviews;
+                expect(body).toBeSortedBy('category', {descending: true});
```

Comment:
Here im not sure you want them to be ordered by category, you only want ones that are social deduction reviews